lib/rollbax/item.ex

```diff
@@ -50,7 +50,13 @@ defmodule Rollbax.Item do
   end
 
   defp exception(:exit, value) do
-    %{"class" => "exit", "message" => Exception.format_exit(value)}
+    message =
+      if Exception.exception?(value) do
+        Exception.format_banner(:error, value)
```

Comment:
Note: this is good™ because we get the `** (ExceptionName)` part as well since on Rollbar it would still have "exit" as the "class" of the exception:

<img width="240" alt="screen shot 2017-02-01 at 12 57 57 pm" src="https://cloud.githubusercontent.com/assets/3890250/22505924/172fc4e6-e87e-11e6-8415-e47c5707622a.png">